src/ORM/Table.php

```diff
@@ -121,6 +121,13 @@ class Table implements RepositoryInterface, EventListenerInterface
     use EventManagerTrait;
 
     /**
+     * Name of default validation set.
+     *
+     * @var string
+     */
+    const VALIDATOR_DEFAULT = 'default';
```

Comment:
I usually name things `SUBJECT_DESCRIPTOR`, where `SUBJECT` is the *thing* the `DESCRIPTOR` applies to.